inc/saitypes.h

```diff
@@ -1196,6 +1199,54 @@ typedef struct _sai_port_snr_list_t
     sai_port_snr_values_t *list;
 } sai_port_snr_list_t;
 
+/**
+ * @brief poe port active channel (when delivering power)
+ */
+typedef enum _sai_poe_port_active_channel_id_t
+{
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_A = 0,
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_B = 1,
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_A_AND_B = 2
+} sai_poe_port_active_channel_id_t;
```

Comment:
either rename the typedef to: sai\_poe\_port\_active\_channel\_type\_t  or the enums to SAI\_POE\_PORT\_ACTIVE\_CHANNEL\_ID\_A

Reply:
Done, thanks.